apps/vscode/command_client/visual_studio.py

```diff
@@ -0,0 +1,15 @@
+from talon import Context, actions
+
+ctx = Context()
+
+ctx.matches = r"""
+app: visual_studio
+"""
+
+ctx.tags = ["user.command_client"]
+
+
+@ctx.action_class("user")
+class VisualStudioActions:
+    def command_server_directory() -> str:
+        return "visual-studio-commandServer"
```

Comment:
Why the mixed format here?  I would argue for consistency with what we do for vscode:
```suggestion
        return "visual-studio-command-server"
```

Reply:
Can you commit that for me, the location change seems fine to me.